transkribus_fixer/cli.py

```diff
@@ -1,26 +1,43 @@
-from click import command, Choice, argument, option
+from pkg_resources import resource_filename
+from click import command, Choice, File, argument, option
 from lxml import etree as ET
 
 from .fixer import TranskribusFixer, NS
 
-@command()
-@option('-f', '--fixes', help="Fixes to apply. Repeatable", type=Choice(['reading_order', 'table', 'metadata', 'namespace']), multiple=True)
-@argument('input-file', nargs=1)
-@argument('output-file', nargs=1)
-def cli(fixes, input_file, output_file):
-    """
-    Apply the fixes to INPUT_FILE.
+FIXERS = [func[4:] for func in dir(TranskribusFixer)
+          if callable(getattr(TranskribusFixer, func)) and func.startswith('fix_')]
+FIXERDOCS = [func + ': ' + getattr(TranskribusFixer, 'fix_' + func).__doc__ for func in FIXERS]
```

Comment:
I appreciate how you reuse the docstrings, also in core for the CLI, neat pattern.

Reply:
It's really just being lazy and trying to make the CLI see the existing API documentation, which increases the chance of investing some time into a decent docstring.

Note: you'd also have to replace `fix_` with `convert_` here when you merge with `rename`.

Reply:
It might be DRY-lazy but it also gives an incentive to write documentation because it's actually used :)

> Note: you'd also have to replace fix_ with convert_ here when you merge with rename.

Yes, and change the `4:` to `8:`.